web/app/js/components/PageHeader.jsx

```diff
@@ -0,0 +1,57 @@
+import _ from 'lodash';
+import React from 'react';
+import { Col, Radio, Row } from 'antd';
+
+export default class PageHeader extends React.Component {
+  constructor(props) {
+    super(props);
+    this.onTimeWindowClick = this.onTimeWindowClick.bind(this);
+    this.state = {
+      selectedWindow: this.props.api.getMetricsWindow()
+    };
+  }
+
+  onTimeWindowClick(e) {
+    let window = e.target.value;
+    this.props.api.setMetricsWindow(window);
+    this.setState({selectedWindow: window});
+  }
+
+  renderMetricWindowButtons() {
+    // don't use time window changing until the results of Telemetry Scalability are in
+    // https://github.com/runconduit/conduit/milestone/4
+    if (this.props.hideButtons) {
```

Comment:
I was initially a bit confused by this comment, since I thought it applied to line 23 below (the `this.props.hideButtons` check). But it really applies to this method as a whole. In that case I might consider moving the comment to where the method call is commented out, on line 50 of this file.

Reply:
what do you think about moving the comment outside (just above) the method instead?

Reply:
Cool, yeah, that works for me too.